package.json

```diff
@@ -18,40 +18,41 @@
   "author": "",
   "license": "MIT",
   "devDependencies": {
+    "babel-eslint": "^8.2.2",
```

Comment:
Why did you reorder this?
- all dependencies from the ember stack
- all third party dependencies
- all our own dependencies

this order makes it easier while doing an ember update

Reply:
We started to re-order, and set it alphabetically. I still like to have our own dependencies (especially the ones that are private) to put at the end

Reply:
I left it ordered alphabetically and put our own dependencies at the end